components/gitpod-protocol/src/protocol.ts

```diff
@@ -779,7 +779,7 @@ export namespace ExternalImageConfigFile {
 
 export interface WorkspaceContext {
     title: string;
-    normalizedContextURL?: string;
+    normalizedContextURL?: string; // TODO: make this NOT optional
```

Comment:
IMO a comment ala `/* This contains the URL portion of the contextURL (which might contain other modifiers as well). It's optional because it's not set for older workspaces.' */